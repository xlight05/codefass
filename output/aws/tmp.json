{
  "Comment": "Test Desc",
  "StartAt": "ChoiceBlock",
  "States": {
    "sequenceOne": {
      "Type": "Task",
      "Resource": "${sequenceOneArn}",
      "Next": "sequenceTwo"
    },
    "sequenceTwo": {
      "Type": "Task",
      "Resource": "${sequenceTwoArn}",
      "End": true
    },
    "par": {
      "Type": "Parallel",
      "End": true,
      "Branch": [
        {
          "StartAt": "parallelOne",
          "States": {
            "parallelOne": {
              "Type": "Task",
              "Resource": "${parallelOneArn}",
              "End": true
            }
          }
        },
        {
          "StartAt": "parallelTwo",
          "States": {
            "parallelTwo": {
              "Type": "Task",
              "Resource": "${parallelTwoArn}",
              "End": true
            }
          }
        }
      ]
    },
    "ChoiceBlock": {
      "Type": "Choice",
      "Choices": [
        {
          "Variable": "$.type",
          "Next": "sequenceOne",
          "StringEquals": "seq"
        }
      ],
      "Default": "par"
    }
  }
}
