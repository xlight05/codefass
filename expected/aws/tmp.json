{
  "Comment": "Auto generated serverless workflow from FaaS",
  "StartAt": "ChoiceState",
  "States": {
    "ChoiceState": {
      "Type": "Choice",
      "Choices": [
        {
          "Variable": "$.type",
          "StringEquals": "seq",
          "Next": "sequenceOne"
        }
      ],
      "Default": "par"
    },
    "sequenceOne": {
      "Type": "Task",
      "Resource": "${lambdaArn1}",
      "Next": "sequenceTwo"
    },
    "sequenceTwo": {
      "Type": "Task",
      "Resource": "${lambdaArn2}",
      "End": true
    },
    "par": {
      "Type": "Parallel",
      "End": true,
      "Branches": [
        {
          "StartAt": "parallelOne",
          "States": {
            "parallelOne": {
              "Type": "Task",
              "Resource": "${lambdaArn3}",
              "End": true
            }
          }
        },
        {
          "StartAt": "parallelTwo",
          "States": {
            "parallelTwo": {
              "Type": "Task",
              "Resource": "${lambdaArn4}",
              "End": true
            }
          }
        }
      ]
    }
  }
}
